src/Listener/ViewListener.php

```diff
@@ -479,6 +479,11 @@ protected function _getAllowedActions()
 
         $blacklist = (array)$this->_action()->getConfig('scaffold.actions_blacklist');
         $blacklist = array_combine($blacklist, $blacklist);
+        foreach ($this->_crud()->getConfig('actions') as $action => $config) {
```

Comment:
Wondering if we can we avoid the `foreach` loop and directly blacklist `Crud.Lookup`? Even if it's not configured/enabled having it in blacklist won't do any harm.

Reply:
How would you do that? The lookup action could be mapped to something else so this is imo the best way.

Reply:
Okay, makes sense.